### PR TITLE
Version control with semvers

### PR DESCRIPTION
**Pull request by ligenxxxx**
- Use semver.
- VTX_NAME align left.
- Modify debug information when VTX power up.
- Vtx version number transfer to VRX is being verified.

### CODE REVIEW

File: src/msp_displayport.c [hunk 1]
```diff
@@ -421,17 +422,21 @@ uint8_t get_tx_data_5680() // prepare data to VRX
...
 
     tx_buf[15] = cam_4_3 ? 0xaa : 0x55;
 
-    tx_buf[16] = 0x40; // crc
+    tx_buf[16] = VTX_VERSION_MAJOR;
+    tx_buf[17] = VTX_VERSION_MINOR;
+    tx_buf[18] = VTX_VERSION_PATCH_LEVEL;
 
-    return 17;
+    tx_buf[19] = 0x40; // crc
```
Comment (cruwaller) [inline src/msp_displayport.c:437]: CRC is still the same?

File: src/common.h [hunk 2]
```diff
@@ -60,7 +57,7 @@
 //#define FIX_EEP
 
 #ifndef _RF_CALIB
-//#define _DEBUG_MODE
+#define _DEBUG_MODE
```
Comment (cruwaller) [inline src/common.h:60]: Should be reverted

File: src/msp_displayport.c (hunk 1 above)
Comment (ligenxxxx) [reply src/msp_displayport.c]: Useless

Comment (cruwaller) [reply src/msp_displayport.c]: Yes, just to avoid confusion. Thanks

File: src/version.h [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#ifndef __VERSION_H_
+#define __VERSION_H_
+
+#define _STR(x)                 #x
+#define STR(x)                  _STR(x)
+#define VTX_VERSION_MAJOR       0 // increment when a major release is made (big new feature, etc)
+#define VTX_VERSION_MINOR       1 // increment when a minor release is made (small new feature, change etc)
+#define VTX_VERSION_PATCH_LEVEL 0 // increment when a bug is fixed
```
Comment (bkleiner) [inline src/version.h:8]: nitpick: i would move those the the beginning of the files as the are the only thing user adjustable here

**Review by bkleiner — APPROVED** [review]
Looks good! now it would be nice if we find a good way to feed the version in form a git tag.

**Review by cruwaller — APPROVED** [review]

**Comment by ligenxxxx** [comment]
log

**Review by bkleiner — APPROVED** [review]

**Pull request merged**